datadog_checks_dev/datadog_checks/dev/tooling/commands/env/edit.py

```diff
@@ -0,0 +1,28 @@
+# (C) Datadog, Inc. 2018-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import click
+
+from datadog_checks.dev.tooling.e2e.config import locate_config_file
+
+from ....fs import file_exists
+from ...testing import complete_envs
+from ...utils import complete_testable_checks, get_tox_file
+from ..console import CONTEXT_SETTINGS, abort, echo_failure
+
+
+@click.command(context_settings=CONTEXT_SETTINGS, short_help='Edit config file using default editor')
+@click.argument('check', autocompletion=complete_testable_checks)
+@click.argument('env', autocompletion=complete_envs)
+@click.option('--editor', '-e', help='Editor to use')
+@click.pass_context
+def edit(ctx, check, env, editor):
+    """Start an environment."""
+    if not file_exists(get_tox_file(check)):
+        abort(f'`{check}` is not a testable check.')
+
+    config_file = locate_config_file(check, env)
```

Comment:
It's often the case that we have a single env running at a time, if so, can we use the underlying autocomplete functions to determine which check and env that is, and if there is only one, just use those values?

That would shorten the command to `ddev env edit` which would be even easier to use.

Reply:
Seems a good idea to shorten the command.

> underlying autocomplete functions
Any pointer how to do that ? 

I think it's also applicable to some other commands like `ddev env stop` and `ddev env shell`.

Reply:
Agree, adding this shortener to those other commands could be useful too.

[`get_active_checks`](https://github.com/DataDog/integrations-core/blob/8cdaad622b02a59f949b84882808974aab715fb5/datadog_checks_dev/datadog_checks/dev/tooling/e2e/config.py#L62) and [`get_configured_envs`](https://github.com/DataDog/integrations-core/blob/8cdaad622b02a59f949b84882808974aab715fb5/datadog_checks_dev/datadog_checks/dev/tooling/e2e/config.py#L40) should be the functions to check.  If they return single values, then 👍 , otherwise return error.

Reply:
@mgarabed Thanks.

> get_active_checks and get_configured_envs

I'm wondering if those are reliable way to get the current "active checks" and envs ?

I'm currently running `snmp` integration with env `py38-python_ad`:

```
§ docker ps
CONTAINER ID   IMAGE                         COMMAND                  CREATED       STATUS                 PORTS                    NAMES
85c7fba3a9c4   datadog/agent:7.28.0-rc.1     "/init"                  4 hours ago   Up 4 hours (healthy)                            dd_snmp_py38-python_ad
8a8600a3e42d   datadog/docker-library:snmp   "entrypoint.sh --age…"   4 hours ago   Up 4 hours             0.0.0.0:1161->1161/udp   dd-snmp
```

But `get_active_checks` is returning much more.

```
 ['activemq', 'activemq_xml', 'aerospike', 'airflow', 'apache', 'btrfs', 'cacti', 'cassandra', 'cassandra_nodetool', 'ceph', 'cilium', 'clickhouse', 'confluent_platform', 'consul', 'coredns', 'couch', 'directory', 'disk', 'dns_check', 'druid', 'elastic', 'envoy', 'etcd', 'fluentd', 'gearmand', 'gitlab', 'gitlab_runner', 'go_expvar', 'gunicorn', 'haproxy', 'harbor', 'hazelcast', 'hive', 'http_check', 'ibm_db2', 'ibm_mq', 'ibm_was', 'ignite', 'istio', 'jboss_wildfly', 'kafka', 'kafka_consumer', 'kong', 'kyototycoon', 'lighttpd', 'linkerd', 'marklogic', 'mesos_master', 'mesos_slave', 'mongo', 'myjmx11', 'mysql', 'nagios', 'neo4j', 'network', 'nginx', 'openmetrics', 'openstack_controller', 'oracle', 'pgbouncer', 'postgres', 'presto', 'process', 'prometheus', 'proxysql', 'rabbitmq', 'redisdb', 'resin', 'riak', 'sap_hana', 'scylla', 'snmp', 'solr', 'spark', 'sqlserver', 'squid', 'ssh_check', 'statsd', 'supervisord', 'systemd', 'tcp_check', 'tls', 'tokumx', 'tomcat', 'twemproxy', 'varnish', 'vault', 'vertica', 'vsphere', 'yarn', 'zk']
```

I think it's listing all the old envs I ran before, `get_active_checks` seems to be based on my local `/Users/<USERNAME>/Library/Application Support/dd-checks-dev/envs`.

I would more expect `get_active_checks` (or similar method) to return only the `snmp` with `py38-python_ad`. Maybe we can do that by check active docker containers instead that seems more reliable.

It seems making `ddev env edit`, `ddev env stop` and `ddev env shell` use current active check/env without argument might be more appropriate as a separate feature/PR. WDYT ?